app/views/admin/resource/index.html.erb

```diff
@@ -1 +1,8 @@
-<%= render "index" %>
+<div class="wrap">
+  <%= render partial: "/shared/sidebar" %>
+  <div class="col-xs-8">
```

Comment:
is there an assumption here about the number of columns and the number of attributes rendered in the table? If there is, we should probably make that class selector dynamic, if not, seems OK. I'm just not that familiar with bootstraps grid system.